build.gradle

```diff
@@ -21,14 +21,14 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.3'
+        classpath 'com.android.tools.build:gradle:4.0.0'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
-        classpath "com.github.laimiux:gradle-android-junit-jacoco-plugin:a910712557"
+        classpath "com.vanniktech:gradle-android-junit-jacoco-plugin:0.16.0"
```

Comment:
Maybe this caused the issue with jacoco coverage?

Reply:
There is a different error we get if we update the other plugins and don't update this one too:

<img width="1533" alt="Screen Shot 2020-06-25 at 3 32 58 PM" src="https://user-images.githubusercontent.com/1459320/85792514-3cb79e80-b6f9-11ea-84c1-90d321a090cb.png">